test/run/ch-fromhost.bats

```diff
@@ -34,11 +34,24 @@ fromhost_ls () {
     find "$1" -xdev -name '*sotest*' -ls
 }
 
+glibc_version_ok () {
+    # Ensure host glibc and guest glibc are compatible (see #1430).
+    host=$(ldd --version | head -1 | grep -o '....$')
```

Comment:
A few suggestions here:

1. Consider how to re-write this regex so it doesn't assume a specific number of characters in the version number, but rather the content.
2. You can do this with a single `sed` invocation, but `head` and `grep` isn't horrible.
3. If you were to stick with `grep`, use `grep -E` for extended regex syntax (and because `egrep` will make ShellCheck yell at you).